cybot/settings.py

```diff
@@ -1,13 +1,13 @@
-import sys
+import os
 
 # find the token and parse it
 TOKEN_PATH = 'token.txt'
-with open(TOKEN_PATH) as TOKEN_SOUCRE:
-    TOKEN = TOKEN_SOUCRE.read().strip('\n')
+with open(TOKEN_PATH) as TOKEN_SOURCE:
+    TOKEN = TOKEN_SOURCE.read().strip('\n')
+
+DIR = os.path.dirname(__file__) + '/'
+CMD_SIGN = '$'
 
 user_kick_timeout = 700
 eng = "poiuytrewqlkjhgfdsamnbvcxz"
 heb = "פםןוטארק'/ךלחיעכגדשצמנהבסז"
-
-with open('help_file.txt') as help_file:
-    DOC_TXT = help_file.read()
```

Comment:
where do we load it? please explain the changes

Reply:
the description of the command taken from the __doc__ of the function so the help file is unneeded